ApiTests/Tests/API/APITests.cs

```diff
@@ -1,44 +1,58 @@
+using RestSharp;
+using System.Net;
 using TestProject.Models;
 using TestProject.Utils;
 
+
 namespace TestProject.Tests.API
 {
     public class Tests
     {
         [SetUp]
         public void Setup()
-        {
-            //move test data creation here
-        }
-
-        [Test]
-        public void PetTest()
         {
             //create a pet using post request
+            //move test data creation here
             PetStoreApiUtils.PostPet(
                 new Pet(
                     ConfigReader.GetNumericalTestDataValue("petId"),
                     ConfigReader.GetTestDataValue("petName"),
                     ConfigReader.GetTestDataValue("petStatus")));
+        }
 
+        [Test]
+        public void PetTest()
+        {
             //validate that the name of the pet is as you passed in a previous step
-
-            //update pet and change the name to a new one and validate that the request was successful
-
-            //validate that the name of the pet is updated to a new one
             Assert.That(PetStoreApiUtils.GetPetById(
                 ConfigReader.GetNumericalTestDataValue("petId")).Name,
-                Is.EqualTo(ConfigReader.GetTestDataValue("newPetName")),
-                "Pet name is not as expected");
+                Is.EqualTo(ConfigReader.GetTestDataValue("petName")), "Pet name is not as expected");
 
-            //delete a pet from the petstore
-            PetStoreApiUtils.DeletePetById(ConfigReader.GetTestDataValue("petId"));
+            //update pet and change the name to a new one and validate that the request was successful
+            RestResponse updateResponse = PetStoreApiUtils.PutPet(
+                new Pet(
+                    ConfigReader.GetNumericalTestDataValue("petId"),
+                    ConfigReader.GetTestDataValue("newPetName"),
+                    ConfigReader.GetTestDataValue("petStatus")));
+
+            Assert.Multiple(() =>
+            {
+                Assert.That(updateResponse.StatusCode, Is.EqualTo(HttpStatusCode.OK), "Pet update request was not successful");
```

Comment:
would it make sense to check any response values if status code is not OK? 
I believe that separating these 2 assertions would be beneficial for test execution time in some cases, as we won't have to spend time on validating the fields if response is not ok